src/pages/explore/PayoutsPage/components/PayoutsList.tsx

```diff
@@ -80,6 +97,21 @@ const PayoutsList = ({ api, members }: PayoutsListProps): JSX.Element => {
 
   if (members.length === 0) return <>No members</>
 
+  const onClainPayout = () => {
+    const tx = api.tx.society.payout()
+
+    const notReallySure = () => {
+      console.log('oiii ??????')
+    }
+
+    payout(tx, api, activeAccount, notReallySure)
+  }
+
+  members[members.length - 1] = {
+    ...members[members.length - 1],
+    hasPayouts: true
+  }
```

Comment:
Ignore, will be removed, just forcing a payout

obs: how to make it better using `config.yml`?